Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM alpine/git AS sources
+
+RUN git clone --depth=1 --branch=config-sessions https://github.com/onaio/express-server.git /usr/src/express-server
```

Comment:
use master branch

Reply:
@KipSigei @ciremusyoka why not create a release tag and use that?

Reply:
@moshthepitt does this mean i have to find a way of passing the tag when building the image and maybe in case a tag is not specified default to master?

Reply:
It means you'll replace `--branch=config-sessions` with `--branch=your-tag-here`